draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -256,8 +256,8 @@ resolved if the remote endpoint is too many segments away.
 
 When mDNS fails, ICE will attempt to fall back to either NAT hairpin,
 if supported, or TURN relay, if not. As noted in {{IPHandling}}, this may
-result in increased media latency and reduced connectivity/increased cost
-(depending on whether the application chooses to use TURN).
+result in increased media latency, reduced connectivity, reduced throughput
+and increased cost (depending on whether the application chooses to use TURN).
```

Comment:
or?

Reply:
I think this needs to be something like  "increased media latency, increased cost, and reduced throughput (if the application chooses to use TURN), or reduced connectivity (if not)."

Reply:
I was thinking that, in theory, hairpinning could also increase latency and reduce throughput.
Agreed that reduced connectivity should probably be separated.
I'll update to make things more precise.

Reply:
> I was thinking that, in theory, hairpinning could also increase latency and reduce throughput.

Not only in theory. This happens a lot in cable networks due to DS-Lite Gateways.

Reply:
This seems kind of wordy now. I don't think we need to be super precise, we can just list the things as potential problems without getting into all the minutiae.

Happy to take a shot at it if useful.

Reply:
New shorter version